MIGRATION.portable-stories.md

```diff
@@ -0,0 +1,368 @@
+  - [What will you achieve at the end of this migration?](#what-will-you-achieve-at-the-end-of-this-migration)
+  - [Getting started](#getting-started)
+    - [1 - Import project-level annotations from Storybook](#1---import-project-level-annotations-from-storybook)
+    - [2 - Configure your testing framework to use portable stories](#2---configure-your-testing-framework-to-use-portable-stories)
+      - [Vitest](#vitest)
+      - [Jest](#jest)
+    - [3 - Remove storyshots from your project](#3---remove-storyshots-from-your-project)
+    - [4 - (Optional) Extend your testing coverage](#4---optional-extend-your-testing-coverage)
+    - [5 - Provide feedback](#5---provide-feedback)
+
+## Pre-requisites
+
+Before you begin the migration process, ensure that you have:
+
+- [ ] A Storybook project with `@storybook/react` or `@storybook/vue3`.
+- [ ] A working Storybook setup with version 7.
+- [ ] Familiarity with your current Storybook and its testing setup.
+
+## What are portable stories?
+
+Storybook provides a `composeStories` utility that assists in converting stories from a story file into renderable elements that can be reused in your Node tests with JSDOM. It also makes sure to apply all their necessary decorators, args, etc., so your component can render correctly. We call these portable stories.
+
+Currently, the only available renderers that provide this functionality are React and Vue3. We have plans to implement this for other renderers soon. If you are using a different renderer (e.g., Angular, Svelte), we recommend that you follow the [test-runner migration](./MIGRATION.test-runner.md) instead.
+
+## What will you achieve at the end of this migration?
+
+If you want to have a similar experience you had with the Storyshots addon, portable stories can help you achieve that. With it, you still have a single test file that can run in a JSDOM environment, rendering all your stories and snapshotting them. However, you may run into similar limitations as you had with the Storyshots addon:
+
+- You are not testing against a real browser.
+- You must mock many browser utilities (e.g., canvas, window APIs, etc).
+- Your debugging experience will not be as good, given you can't access the browser as part of your tests.
+
+Alternatively, you may want to consider migrating to the [test-runner](./MIGRATION.test-runner.md), which is more powerful, runs against a real browser with [Playwright](https://playwright.dev/), provides multi-browser support, and more.
+
+## Getting started
+
+We recommend you turn off your current storyshots tests to start the migration process. To do this, rename the configuration file (i.e., `storybook.test.ts` or similar) to a different name. This will prevent the tests from being detected, as you'll be creating a new testing configuration file with the same name. By doing this, you'll be able to preserve your existing tests while transitioning to portable stories.
```

Comment:
@jonniebigodes the reason I asked people to rename `storybook.test.ts` to `storybook.test.ts.old` was so that the test was not detected anymore. If we just tell them to rename to something else, the tests will still run and that will collide with the migration